deps/wazuh_testing/wazuh_testing/remote.py

```diff
@@ -13,11 +13,97 @@
 TCP_UDP = "TCP,UDP"
 
 
+def callback_detect_syslog_allowed_ips(syslog_ips):
+    msg = fr"Remote syslog allowed from: \'{syslog_ips}\'"
+    return make_callback(pattern=msg, prefix=REMOTED_DETECTOR_PREFIX)
+
+
+def callback_detect_syslog_denied_ips(syslog_ips):
+    msg = fr"Message from \'{syslog_ips}\' not allowed. Cannot find the ID of the agent."
+    return make_callback(pattern=msg, prefix=REMOTED_DETECTOR_PREFIX)
+
+
+def callback_invalid_value(option, value):
+    msg = fr"ERROR: \(\d+\): Invalid value for element '{option}': {value}."
+    return make_callback(pattern=msg, prefix=REMOTED_DETECTOR_PREFIX)
+
+
+def callback_error_in_configuration(severity):
+    msg = fr"{severity}: \(\d+\): Configuration error at '/var/ossec/etc/ossec.conf'."
```

Comment:
This should be changed to use the variable from the `wazuh_testing.tools` module.